RF24.h

```diff
@@ -1867,6 +1909,18 @@ class RF24 {
      * @param qty How many successive registers to print
      */
     void print_address_register(const char* name, uint8_t reg, uint8_t qty = 1);
+    /**
+     * Put the value of a 40-bit address register in a char array
+     *
+     * Optionally it can print some quantity of successive
+     * registers on the same line.  This is useful for printing a group
+     * of related registers on one line.
+     *
+     * @param ^out_buffer output buffer
```

Comment:
the `^` seems like a typo, but again you don't need an indication that this is passed by reference (doxygen shows that in the member prototype).

Reply:
fixed